setup.cfg

```diff
@@ -93,6 +93,7 @@ docs =
 [options.package_data]
 * = data/*, data/*/*, data/*/*/*, data/*/*/*/*, data/*/*/*/*/*, data/*/*/*/*/*/*
 astropy = astropy.cfg, CITATION
+astropy.cosmology = realizations.ecsv
```

Comment:
The file itself does not seem to be included in the PR. Also, a possible problem if that becomes our primary source: we would then need to depend on YAML.

Reply:
Thanks! that's actually a relic of the old PR direction. I've removed it.  
I now have a working implementation of auto-import of all user-defined realizations in any file matching the glob `~/.astropy/cosmology/*.ecsv`.

<img width="1087" alt="Screen Shot 2021-04-13 at 21 33 11" src="https://user-images.githubusercontent.com/8949649/114641300-dad41f80-9c9f-11eb-8641-b03f041a13ad.png">

Reply:
I've added WIP to the title, until the spaghetti code is cleaned up.